CourseManager/Courses/Course 1 Build a SwiftUI app for iOS 15/Components/BlobView.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  BlobView.swift
+//  SwiftUIPractice1
+//
+//  Created by Summer Crow on 2022-09-27.
+//
+
+import SwiftUI
+
+struct BlobView: View {
+    @State var appear = false
+    var body: some View {
+        // BlobShape()
```

Comment:
I guess you should not merge code with comments in it

Reply:
yup, you're right missed this one, thanks